gemfiles/rails7.gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-rails_version = '~> 7.0.0'
+rails_version = '~> 7.0'
```

Comment:
I think, we need to split this gemfile into two, one for ~> 7.0.0 series and one for ~> 7.0, since the differences are significant between 7.0 and 7.2

Reply:
Usually, I'd think it necessary but since Logidze has additional exposure from using non-public APIs from Rails, it's a good idea!